multicluster/config/default/kustomization.yaml

```diff
@@ -0,0 +1,74 @@
+# Adds namespace to all resources.
+namespace: antrea-system
```

Comment:
i think this is still an open discussion, whether to have `antrea-system` as a separate namespace. i feel like all antrea components should be in same namespace, odd to have multi-cluster antrea namespace and have controller agents in `kube-system`..
i don't know if `multicluster-antrea-system` namespace is an acceptable compromise ?

Reply:
it thought it was settled we create antrea-system.

But again, u guys would need to decide fast if it is changed again.

Reply:
ok.. if majority feels that way .. im fine with it..